src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -221,6 +253,19 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                 Return valueText
             End Function
 
+            Private Function GetIntegerLiteralValueString(value As Long, base As LiteralBase) As String
+                Select Case base
+                    Case LiteralBase.Decimal
+                        Return value.ToString()
+                    Case LiteralBase.Hexadecimal
+                        Return $"&H{Convert.ToString(value, 16).ToUpperInvariant()}"
+                    Case LiteralBase.Octal
+                        Return $"&O{Convert.ToString(value, 8)}"
+                    Case Else
+                        Return ""
```

Comment:
@jasonmalinowski - I might be missing something but from what I can see Contract.Unreachable only exists in the Microsoft.VisualStudio.InteractiveWindow namespace which is not accessible from this project. Should that be ExceptionUtilities.Unreachable instead?